Clarinet.toml

```diff
@@ -29,13 +32,18 @@ depends_on = ["citycoin-auth", "citycoin-token", "citycoin-vrf", "citycoin-core-
 path = "contracts/clarinet/citycoin-core-v2.clar"
 depends_on = ["citycoin-auth", "citycoin-token", "citycoin-vrf", "citycoin-core-trait"]
 
-# contracts listed below are used only in testing
+[contracts.citycoin-core-mal]
+path = "contracts/clarinet/citycoin-core-v2.clar"
+depends_on = ["citycoin-auth", "citycoin-token", "citycoin-vrf", "citycoin-core-trait"]
```

Comment:
What's `ma1` stands for?

Reply:
Malicious, based off the audit report was using the same language, it's basically a core contract that gets caught in the wrong state. Could just call it v2 if that makes more sense.

Reply:
In that case can we use full "malicious" word instead of abbreviation? That way it will be clear for everyone what it is.

Reply:
I think I"m just going to rename it `-v3.clar` since the test code uses `core3` as the reference, and it's only used in one test case. It's not technically malicious it's just testing incorrect state references, so that may be too misleading anyway.

Reply:
Backticks broke my comment but fixed in https://github.com/citycoins/citycoin/pull/162/commits/7b3920851226d799b98d8b2216c4ff06900e5ce0

Reply:
OK, sounds good